lib/application.js

```diff
@@ -225,6 +225,40 @@ Application.prototype.addCommands = function () {
     })
   })
 
+  /**
+   * Utility from webdriverio v5
+   * https://github.com/webdriverio/webdriverio/blob/v5.9.4/packages/webdriverio/src/commands/browser/switchWindow.js
+   */
+  this.client.addCommand('switchWindow', async function (urlOrTitleToMatch) {
+    if (typeof urlOrTitleToMatch !== 'string' && !(urlOrTitleToMatch instanceof RegExp)) {
+      throw new Error('Unsupported parameter for switchWindow, required is "string" or a RegExp')
```

Comment:
Let's throw a `TypeError` here and say `Invalid parameter urlOrTitleToMatch: expected a string or a RegExp` for clarity.